packages/rainbowkit/src/transactions/TransactionStoreContext.tsx

```diff
@@ -20,13 +26,26 @@ export function TransactionStoreProvider({
   const provider = usePublicClient() as PublicClient;
   const { address } = useAccount();
   const chainId = useChainId();
+  const { refetch } = useBalance({
+    address,
+    query: {
+      enabled: false,
+    },
+  });
```

Comment:
@magiziz Should we move this logic to `useProfile` so that we're caching it and reduce the number of balance fetches? Currently I believe this hook would be called even if showBalance is disabled, and would cause duplicate balance fetch calls when rendering Profile -> Transaction UI